src/Rules/MarkdownWithContent.php

```diff
@@ -23,14 +25,6 @@ public function passes($attribute, $value)
         return strlen($text) > 0;
     }
 
-    public function stripZeroWidthSpaces(string $text): string
```

Comment:
does the markdown rule still strip zerowidth spaces or is this now completely separate and should be added to any project that used to rely on the markdown rule for including it?

Reply:
@ItsANameToo it's used a few lines up [here](https://github.com/ArkEcosystem/laravel-ui/pull/544/files#diff-4cd939532f9a045b1e43016512fe131d9c4ed14010763cc967a9d606a9c63828R23) still, I just moved the method so it could be used solely as its own rule too